src/main/kotlin/onm/house/devices/Kettle.kt

```diff
@@ -0,0 +1,27 @@
+package onm.house.devices
+
+import onm.configuration.DeviceType
+import onm.events.IEventHandler
+import onm.events.WasherDoneEvent
+import java.util.*
+
+class Kettle(override val id: UUID, eventHandler: IEventHandler) : AbstractDevice(DeviceType.KETTLE) {
+
+    private val event = WasherDoneEvent(eventHandler)
+
+    /**
+     * Starts washing clothes. This produces event which is raised after given time period. Note that new thread is created.
+     * */
+    fun startWarmingWater(periodInMinutes: Double) {
```

Comment:
I personally think that kettle does not get time period... it just notifies user when watter is ready